config/betterquesting/DefaultQuests.json

```diff
@@ -189036,7 +189036,7 @@
           "lockedProgress:1": 0,
           "autoClaim:1": 0,
           "isSilent:1": 0,
-          "desc:8": "The Piston Boots are a bouncy pair of shoes for adventurers.\n\nIt functions as a piece of armor for the foot slot that gives a fun jump boost! They let the wearer jump up to THREE blocks in height and increase run speed. They also grant a Step Up enchantment to allow the wearer to step up a single block without jumping while sprinting.\n\nBasically, make them ASAP."
+          "desc:8": "The Piston Boots are a bouncy pair of shoes for adventurers.\n\nIt functions as a piece of armor for the foot slot that gives a fun jump boost! They let the wearer jump up to THREE blocks in height and increase run speed. They also grant a Step Up enchantment to allow the wearer to step up a single block without jumping while sprinting. You can repair them with the leather on the anvil.\n\nBasically, make them ASAP."
```

Comment:
You can repair them with the leather on the anvil.->You can repair them with leather using an anvil.

Are you sure it isn't tanned leather? I thought it was.

Reply:
I'm sure, checked this. No problem, correct by your will - it's not for me english is native.

Reply:
Are you not going to fix the grammar?

Reply:
I thought you will. Ok, I'll do.

Reply:
I don't think I can edit other people's PRs. Or at least, I don't know how. And it's way easier for you to make an updated commit than for me to wait for it to be merged and then make a new PR.

Reply:
Ok.